alacritty/src/display/window.rs

```diff
@@ -182,8 +183,11 @@ impl Window {
         window.set_ime_allowed(true);
         window.set_ime_purpose(ImePurpose::Terminal);
 
-        // Set initial transparency hint.
-        window.set_transparent(config.window_opacity() < 1.);
+        // Set initial background opacity and transparency
+        #[cfg(target_os = "macos")]
+        set_titlebar_transparent(&window, config.window_opacity() < 1.);
+        #[cfg(target_os = "macos")]
+        set_background_opacity(&window, config.window_opacity() as _);
```

Comment:
You're not calling `window.set_transparent` here anymore, why is that?

Reply:
Because it's called in `WindowBuilder::with_transparent` instead.

Reply:
But only on macOS. Contrary to Apple's belief, there are other operating systems. :)

Reply:
That was also called before though? Are you suggesting a winit update has removed the necessity for this on all platforms?

Reply:
Yes, `WindowBuilder::with_transparent` nowadays does the exact same thing as `window.set_transparent` (call `setOpaque:`). In the past, there was confusion about this, because `WindowBuilder::with_transparent` set the window background as well (but it no longer does since https://github.com/rust-windowing/winit/pull/3657).